containers/flatpak/beiboot-extra-packages

```diff
@@ -0,0 +1,8 @@
+https://github.com/cockpit-project/cockpit-machines/releases/download/291/cockpit-machines-291.tar.xz
+5eb5d057aafd2200b0844d7a91bc47d365538af9ec800943df696c09445a5dfe
```

Comment:
Hang on -- I'm about to release a new machines with https://github.com/cockpit-project/cockpit-machines/pull/1105 , let's please update to that.

Reply:
Voilà: https://github.com/cockpit-project/cockpit-machines/releases/tag/292